.github/workflows/master.yml

```
@@ -0,0 +1,34 @@
+name: Push to master
+
+on:
+  push:
+    branches:
+      - 'master'
+    paths:
+      - '.github/workflows/feature.yml'
+      - '.github/workflows/develop.yml'
+      - '.github/workflows/release.yml'
+      - '.github/workflows/master.yml'
+      - '.github/workflows/vere.yml'
+      - 'pkg/arvo/**'
+      - 'pkg/docker-image/**'
+      - 'pkg/ent/**'
+      - 'pkg/ge-additions/**'
+      - 'pkg/libaes_siv/**'
+      - 'pkg/urbit/**'
+      - 'pkg/urcrypt/**'
+      - 'tests/**'
+      - 'bin/**'
+      - 'nix/**'
+      - 'default.nix'
+
+jobs:
+  call-vere:
+    uses: ./.github/workflows/vere.yml
+    with:
+      pace: 'edge'
```

```suggestion
      pace: 'live'
```

The master branch should use the current live version of the binary.